### ISSUE TITLE
[FEATURE REQUEST] CS to CS regridding in gcpy

### ISSUE THREAD

**Issue by lizziel** [geoschem/gcpy#18]
When comparing cubed sphere data sets at different resolutions using compare_single_level I ran into this error:
```
Traceback (most recent call last):
  File "/n/home08/elundgren/bin/compare_var.py", line 20, in <module>
    gcpy.compare_single_level(ds1, desc1, ds2, desc2, varlist=varlist, weightsdir=weightsdir)
  File "/n/home08/elundgren/GC/python/gcpy/gcpy/benchmark.py", line 294, in compare_single_level
    refregridder_list = make_regridder_C2L(refres, cmpres, weightsdir=weightsdir, reuse_weights=True)
  File "/n/home08/elundgren/GC/python/gcpy/gcpy/grid/regrid.py", line 16, in make_regridder_C2L
    llgrid = make_grid_LL(llres_out)
  File "/n/home08/elundgren/GC/python/gcpy/gcpy/grid/horiz.py", line 12, in make_grid_LL
    [dlat,dlon] = list(map(float, llres.split('x')))
AttributeError: 'int' object has no attribute 'split'
```
I traced this to the logic in compare_single_level that determines what regridder to make. The logic is written such that if ref or dev is CS, and the comparison grid is different from the native, a CS to lat-lon regridder is created. However, if both ref and dev are cubed sphere the comparison grid is also cubed sphere, so the CS to lat-lon regridder will not work. Unless argument cmpres is passed such that it is explicitly set as lat-lon, e.g. cmpres='1x1'.

In summary there are two problems: (1) the CS to CS regridding is missing from gcpy (make_regridder_C2C does not exist), and (2) the logic that would call for CS to CS regridding is missing and instead a CS to lat-lon regridder is used, resulting in an error.

To address the second problem I pushed an update to return with an explanatory error message if CS to CS regridding is called for. See https://github.com/geoschem/gcpy/commit/4a2094d2c8de217dc3351442730060942edff1cf.
```
ERROR: CS to CS regridding is not yet implemented in gcpy. Ref and dev cubed sphere grids must be the same resolution, or pass cmpres to compare_single_level as a lat-lon grid resolution.
```
However, this should not be necessary. CS to CS regridding needs to be implemented into gcpy.

**Comment by lizziel** [comment]
This update still needs to happen but in the mean-time there is now handling to default to compare at 1x1.25.

**Comment by msulprizio** [comment]
See fcd06ffb4d7176b111282824d1346dc67e77444a

**Comment by WilliamDowns** [comment]
As of 1.0.0, CS to CS regridding is now available in GCPy's plotting schemes and in the new file regridder.
